identifyConsensus.py

```diff
@@ -10,12 +10,45 @@ def get_parser():
                         nargs="*", help='Paths to HotNet(2) results files.', required=True)
     parser.add_argument('-n', '--networks', nargs="*", required=True,
                         help='List the network that corresponds to each results file.')
+    parser.add_argument('-d', '--delta_choice', action='store_true',
+                        help='Test automated delta selection. Expects a directory of results for each network.')
```

Comment:
Ah, I should change that, had meant for the flag to be used differently, initially.